### PR TITLE
Fix NNCF supported quantization strategies README table 

### PR DESCRIPTION
**Pull request by echarlaix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ Optimum enables the usage of popular compression techniques such as quantization
 
 | Features                           | ONNX Runtime          |     Neural Compressor   |         OpenVINO        |
 |:----------------------------------:|:---------------------:|:-----------------------:|:-----------------------:|
-| Post-training Dynamic Quantization |  :heavy_check_mark:   |    :heavy_check_mark:   |    :heavy_check_mark:   |
+| Post-training Dynamic Quantization |  :heavy_check_mark:   |    :heavy_check_mark:   ||          N/A           |
```
Comment (fxmarty) [inline README.md:63]:
```suggestion
| Post-training Dynamic Quantization |  :heavy_check_mark:   |    :heavy_check_mark:   |          N/A           |
```

**Review by fxmarty — APPROVED** [review]
LGTM

**Comment by fxmarty** [comment]
Dynamic quantization is not supported in OpenVINO?

**Comment by echarlaix** [comment]
> Dynamic quantization is not supported in OpenVINO?

No only post-training static quantization and quantization aware training are supported with NNCF

**Pull request merged**